README.md

```diff
@@ -158,6 +158,7 @@ Full list of options in `config.json`:
 | stage                               | String  | No         | Named external stage name created at pre-requirements section. Has to be a fully qualified name including the schema name. If not specified, table internal stage are used. When this is defined then `s3_bucket` has to be defined as well. |
 | file_format                         | String  | Yes        | Named file format name created at pre-requirements section. Has to be a fully qualified name including the schema name. |
 | batch_size_rows                     | Integer |            | (Default: 100000) Maximum number of rows in each batch. At the end of each batch, the rows in the batch are loaded into Snowflake. |
+| flush_interval_seconds              | Integer |            | (Default: 3600) Maximum time between flushes, if batch_size_rows is not met. |
```

Comment:
Can we make this parameter optional and turn off by default? This is to make sure we keep the current behaviour of the hundreds of other data sources used by this target.